server/app/jobs/OldJobCleanupJob.java

```diff
@@ -0,0 +1,36 @@
+package jobs;
```

Comment:
Suggestion: Put specific implementations in a sub directory so that the infrastructure and concrete jobs are separate

Reply:
Done